libs/design-system/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@novu/design-system",
+  "version": "0.20.0",
+  "repository": "https://github.com/novuhq/novu",
+  "description": "",
+  "scripts": {
+    "start": "npm run build:watch",
+    "prebuild": "rimraf dist",
+    "lint": "eslint --ext .ts,.tsx src",
+    "build": "npm run build:cjs && npm run build:esm && npm run build:types",
+    "build:cjs": "cross-env node_modules/.bin/tsc -p tsconfig.json",
+    "build:esm": "cross-env node_modules/.bin/tsc -p tsconfig.esm.json",
+    "build:esm:watch": "cross-env node_modules/.bin/tsc -p tsconfig.esm.json -w --preserveWatchOutput",
+    "build:types": "tsc --declaration --emitDeclarationOnly --declarationMap --declarationDir dist/types -p tsconfig.json",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
+    "cypress:install": "cypress install",
+    "cypress:open": "cross-env NODE_ENV=test cypress open",
+    "cypress:run": "cross-env NODE_OPTIONS=--max_old_space_size=4096 NODE_ENV=test cypress run --component"
+  },
+  "author": "",
+  "license": "ISC",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "files": [
+    "dist/cjs",
+    "dist/esm",
+    "dist/types"
+  ],
+  "sideEffects": false,
+  "devDependencies": {
+    "@storybook/addon-actions": "^7.5.0",
+    "@storybook/addon-docs": "^7.4.2",
+    "@storybook/theming": "^7.4.2",
+    "@storybook/react": "^7.4.2",
+    "@storybook/react-webpack5": "^7.4.2",
+    "@types/node": "^18.11.12",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "@vitejs/plugin-react": "^4.0.3",
+    "acorn": "^8.7.1",
+    "cypress": "^12.17.2",
+    "eslint-plugin-cypress": "^2.12.1",
+    "eslint-plugin-storybook": "^0.6.13",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-router-dom": "6.2.2",
+    "rimraf": "^3.0.2",
+    "storybook": "^7.4.2",
+    "ts-loader": "~9.4.0",
+    "tslib": "^2.3.1",
+    "typescript": "4.9.5",
+    "url-loader": "^4.1.1",
+    "vite": "^4.4.5",
+    "vite-plugin-dts": "^3.6.0"
+  },
+  "peerDependencies": {
+    "react": ">=16.8.0",
+    "react-dom": ">=16.8.0",
+    "react-router-dom": "^6.10.0"
```

Comment:
Is this the version we finally need? I saw in an earlier file that 6.2.2 was pinned.